### ISSUE TITLE
include steps/* fails with ansible2

### ISSUE THREAD

**Issue by tiawaz** [cbrunnkvist/ansistrano-symfony-deploy#8]
after updating ansible to v2 i see the following failure:

```
TASK [carlosbuenosvinos.ansistrano-deploy : ANSISTRANO | Create softlinks for shared paths] ***
TASK [carlosbuenosvinos.ansistrano-deploy : include] ***************************
included: /etc/ansible/roles/carlosbuenosvinos.ansistrano-deploy/tasks/../../cbrunnkvist.ansistrano-symfony-deploy/config/after_symlink_shared.yml for xxxx

TASK [carlosbuenosvinos.ansistrano-deploy : include] ***************************
fatal: [xxxx]: FAILED! => {"failed": true, "reason": "ERROR! the file_name 'xxxx/ansible/login-deploy/steps/composer.yml' does not exist, or is not readable"}
```

ansible seems to be looking for the steps/* files in the local playbook dir instead of the role dir.

**Comment by tiawaz** [comment]
setting `roles_path = /etc/ansible/roles` in ansible.cfg has no effect

**Comment by tiawaz** [comment]
changing the include statements in `config/after_symlink_shared.yml` to
```
- include: "../../cbrunnkvist.ansistrano-symfony-deploy/config/steps/xxx.yml"
  when: xxx
```

works.

Since installing ansistrano-symfony-deploy in the same parent as ansistrano-deploy is already a requirement a change like this ought to be fine?

**Comment by tiawaz** [comment]
#9

**Comment by tiawaz** [comment]
ping

**Comment by markitosgv** [comment]
+1

**Comment by johnkrovitch** [comment]
+1

**Comment by cbrunnkvist** [comment]
Fixed by #9 it seems 🎯
